Added 19.10 release notes

tmtabor opened the pull request.

tmtabor committed:
Re-adding release notes

bahill commented on releases/index.html:
```diff
@@ -530,6 +542,12 @@ <h3>Notebook Tool Manager 0.1.5 <em style="font-size: 0.8em; color: gray;">(Octo
+                <h3>Rich Text Editor 19.10 <em style="font-size: 0.8em; color: gray;">(October 28, 2019)</em></h3>
+                <ul>
+                    <li>Bug fix for double-clicking erroneously displaying markdown text.</li>
+                    <li>Bug fix for markdwn cells erroneously being changed to code</li>
```
typo - markdwn should be markdown - also need a period at the end to match the others (or take away the period in the previous line)

bahill commented on releases/index.html:
```diff
@@ -106,6 +106,13 @@ <h1>Releases</h1>
+                <h3>GenePattern Notebook Repository 19.10 <em style="font-size: 0.8em; color: gray;">(October 28, 2019)</em></h3>
+                <ul>
+                    <li>Pinned public notebook tags now display an optional description.</li>
+                    <li>Updated to JupyterHub 1.0.</li>
```
We should link out to the doc or announcement for this

bahill replied:
@tmtabor Sorry to be a pain, but Michael wanted us to keep linking out to things like this "JupyterHub 1.0"

bahill commented on releases/index.html:
```diff
@@ -106,6 +106,13 @@ <h1>Releases</h1>
+                <ul>
+                    <li>Pinned public notebook tags now display an optional description.</li>
+                    <li>Updated to JupyterHub 1.0.</li>
+                    <li>Updated to latest versions of genepattern-notebook, nbtools and jupyter-wysiwyg.</li>
```
these should be linked to the relevant sections in this doc for ease of navigation

bahill replied:
Same thing here - Michael's was fine with leaving the release notes layout as is, so long as we linked to the sections of the document in the line that reads "Updated to latest versions" (line 113)

bahill reviewed (changes requested):
There are a few minor edits before this can go out - We should also show this to Michael before announcing, if we are announcing.

tmtabor committed:
Typo fix

bahill commented:
Still a few changes requested- lines 112 & 113

tmtabor committed:
Adding link

bahill committed:
Added links to other components that were also updated

copied from formatting in 19.09 release to allow us to draw attention to these sections without reformatting this page.
